src/test/config.sol

```diff
@@ -1369,8 +1369,8 @@ contract Config {
         });
         afterSpell.collaterals["RWA007-A"] = CollateralValues({
             aL_enabled:   true,
-            aL_line:      250 * MILLION,
-            aL_gap:       50 * MILLION,
+            aL_line:      500 * MILLION,
```

Comment:
I would had a test to ensure the oracle price follows the dc increase in specific spell tests